src/globals/js/components.js

```diff
@@ -16,11 +15,8 @@ export { default as Loading } from '../../components/loading/loading';
 export { default as InlineLoading } from '../../components/inline-loading/inline-loading';
 export { default as Dropdown } from '../../components/dropdown/dropdown';
 export { default as NumberInput } from '../../components/number-input/number-input';
-export { default as DataTable } from '../../components/data-table/data-table';
 export { default as DataTableV2 } from '../../components/data-table-v2/data-table-v2';
```

Comment:
Not sure where we are going, but wanted to see if team wants this is in sync with the latest.